scripts/pull_github_pr.sh

```diff
@@ -24,8 +24,10 @@ done
 NL=$'\n'
 
 PR_NUM=$1
-# convert full repo URL to its project/repo part:
-REMOTE_SLASH_BRANCH="$(git rev-parse --abbrev-ref --symbolic-full-name  @{upstream})"
+# convert full repo URL to its project/repo part, in case of failure default to origin/master:
+REMOTE_SLASH_BRANCH="$(git rev-parse --abbrev-ref --symbolic-full-name  @{upstream} \
+     || git rev-parse --abbrev-ref --symbolic-full-name master@{upstream} \
+     || echo 'origin/master')"
```

Comment:
That behavior is preexisting, but first and foremost, it would only pull the changes locally based on a wrong remote. If you later try to push to the right one, it will fail unless you use `--force`, which you shouldn't. (`--force-with-lease` is the only acceptable way, and that's only for dequeuing, so outside of scope for this script anyway).